components/blog/ui/ArticleContainer.tsx

```diff
@@ -29,16 +30,27 @@ const ArticleContainer = (props: ArticleContainerProps) => {
   const { team } = useParams();
   const { articleListData, thumbnail, description } = props;
   const FilteredCategoryList = useGetCategoryList(team);
+  const CategorySelected = useGetCategory();
 
   if (!FilteredCategoryList) return <LoadingLottie width={10} height={10} fit />;
```

Comment:
요거 useGetCategory의 경우도 갑 존재 여부 체크해줘야할 것 같아요!

```suggestion
  if (!FilteredCategoryList || !CategorySelected) return <LoadingLottie width={10} height={10} fit />;
```